interfaces/cython/cantera/examples/kinetics/custom_reactions.py

```diff
@@ -31,7 +32,9 @@
 
 # old framework - use xml input
 
-gas2 = ct.Solution(mech.replace('.yaml', '.xml'))
+with warnings.catch_warnings():
+    warnings.simplefilter("ignore")
+    gas2 = ct.Solution(mech.replace(".yaml", ".xml"))
```

Comment:
I actually think it'd be better to leave the warning message here, so anyone who copy-pastes this gets that warning and reads about it.

Reply:
This one is unfortunately breaking CI, as the warning is raised to an exception.

Reply:
Ah, that's annoying. I wonder if there's an easy way to fix that... With pytest, it must be possible to run these examples now, possibly just by `import`ing them... then we wouldn't have to do the hacky `find ... -exec ...`... hmm. That'd be a pretty significant scope expansion here though. What do you think?

Reply:
I think I'll add a comment to the example, which is a lot simpler. The speed-test part comparing things to XML will have to be removed after 2.6 regardless - this part of the example was mainly intended as a coarse benchmark test (which btw was highly useful).

Reply:
The `pytest` route is imho a separate topic. As an unrelated note, I've used pytest to cover jupyter notebooks btw, see example [here](https://github.com/ischoegl/ctwrap/blob/main/tests/test_notebooks.py)